packages/core/src/App/Containers/Modals/information-submitted-modal/information-submitted-modal.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Dialog } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { useStore, observer } from '@deriv/stores';
+import './information-submitted-modal.scss';
+
+const InformationSubmittedModal = observer(() => {
+    const { ui } = useStore();
+    const { is_information_submitted_modal_open: is_open, toggleInformationSubmittedModal } = ui;
```

Comment:
Do we need this reassign to is_open

Reply:
it's just renaming to keep a short name inside JSX to match the prop name